package.json

```diff
@@ -88,6 +88,7 @@
     "husky": "^8.0.0",
     "light-my-request": "^5.10.0",
     "lint-staged": "^14.0.1",
+    "mapeo-offline-map": "^2.0.0",
```

Comment:
added as a dev dependency but if we want to be more strict about using this specifically, can change to direct or peer dep.

wasn't too keen on the offline map being bundled as of the npm package so i figured it would make sense to let the consumer decide to add the map as a dependency in its own project